### PR TITLE
feat: gradient accumulation

### PR DESCRIPTION
**Pull request by borisdayma**
allow gradient accumulation

### CODE REVIEW

File: seq2seq/run_seq2seq_flax.py [hunk 1]
```diff
@@ -636,7 +640,14 @@ def decay_mask_fn(params):
         )
 
     # Setup train state
-    state = TrainState.create(apply_fn=model.__call__, params=model.params, tx=optimizer, dropout_rng=dropout_rng)
+    state = TrainState.create(
+        apply_fn=model.__call__,
+        params=model.params,
+        tx=adamw,
```
Comment (khalidsaifullaah) [inline seq2seq/run_seq2seq_flax.py:646]: this should be-
`tx=optimizer`

**Review by khalidsaifullaah — COMMENTED** [review]
the typo followed here as well😁
`wandb.log({f'train/{k}': jax.device_get(v)}, step=global_step)`

File: seq2seq/run_seq2seq_flax.py (hunk 1 above)
Comment (borisdayma) [reply seq2seq/run_seq2seq_flax.py]: Good catch!
